client/jetpack-onboarding/steps/site-type.jsx

```diff
@@ -15,12 +16,24 @@ import PageViewTracker from 'lib/analytics/page-view-tracker';
 import Tile from 'components/tile-grid/tile';
 import TileGrid from 'components/tile-grid';
 import { JETPACK_ONBOARDING_STEPS as STEPS } from '../constants';
+import { saveJetpackOnboardingSettings } from 'state/jetpack-onboarding/actions';
 
 class JetpackOnboardingSiteTypeStep extends React.PureComponent {
+	handleSiteTypeSelection = siteType => {
+		const { siteId } = this.props;
```

Comment:
If you ditched this and instead just inlined `this.props.siteId` below, you could collapse the method definition, using some arrow fun:

```js
handleSiteTypeSelection = siteType => () => {
	this.props.saveJetpackOnboardingSettings( this.props.siteId, {
		siteType,
	} );
}
```

Reply:
Nice, addressed in ec21a1d2e6.